client/src/components/footer/footer.html

```diff
@@ -0,0 +1,15 @@
+<footer layout="row" layout-align="center center">
+  <div class="footer__container"
+      layout="row" layout-align="space-between center">
+    <span>
+      &copy; 2017 //
+      by <a href="https://github.com/hatnote">Hatnote</a> ·
+      <a href="https://github.com/hatnote/montage/blob/master/README.md">About</a>
```

Comment:
Linking to GitHub is good for a `Source` link, but for the About, I think [the Commons page](https://commons.wikimedia.org/wiki/Commons:Montage) may be better.

Reply:
Fixed in e200472239b890c7ab94dcf25de60a7f8e584178.